mailbox/noise.go

```diff
@@ -501,18 +509,14 @@ const (
 	// 1 + 33 + 16
 	ActOneSize = 50
 
-	// ActTwoPayloadSize is the size of the fixed sized payload that can be
-	// sent from the responder to the initiator in act two.
-	ActTwoPayloadSize = 500
-
-	// ActTwoSize is the size the packet sent from responder to initiator
-	// in ActTwo. The packet consists of a handshake version, an ephemeral
-	// key in compressed format, the static key of the responder (encrypted
-	// with a 16 byte MAC), a fixed 500 bytes reserved for the auth
-	// payload, and a 16-byte poly1305 tag.
+	// ActTwoOverheadSize is the minimum size of the packet sent from
+	// responder to initiator in ActTwo. The packet consists of a
+	// handshake version, an ephemeral key in compressed format, the static
+	// key of the responder (encrypted with a 16 byte MAC) and 2 byte field
```

Comment:
nit: 4 bytes, here and below. Total should come to 87 now as well?

Reply:
ah yes! 🙈  thanks!

Reply:
looks like it is still missing something else aswell (itests are hanging) ... looking into it

Reply:
ok cool, found the other bug. was also constant related 🙃 updated! 🚀